packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -408,16 +424,17 @@ const QueryEditorHeaderContent = observer(
                   'btn--icon__caution': isExistingQueryName,
                 })}
                 onClick={renameQuery}
-                title="Rename Query"
+                title="Rename query"
               >
                 <CheckIcon />
```

Comment:
You know, I think we can do a little better, we don't need the check button, on `Enter`, we will save, I think this is a fairly clear interaction